airflow/utils/db.py

```diff
@@ -533,6 +533,13 @@ def create_default_connections(session: Session = NEW_SESSION):
         ),
         session,
     )
+    merge_conn(
+        Connection(
+            conn_id="salesforce_default",
+            conn_type="salesforce",
```

Comment:
I think it would be really nice to have some connection attributes filled in (even with bogus values like the others here) for what _could_ be a "useful" example connection.

Reply:
I have fixed this.